specs/integration/search/CreateDatasetWidget.js

```diff
@@ -0,0 +1,12 @@
+contributeLink: 'a[title="Contribute!"]',
```

Comment:
> You could also use `{ a: "Contribuer" }` as a [location method](https://github.com/MattiSG/Watai/wiki/Widgets#alternative-location-methods) ([example](https://github.com/sgmap/mes-aides-ui/blob/rewrite/test/integration/HomepageWidget.js#L3)).

Use what's the most likely to be stable: either the title attribute or part of the text content.